client/bash.exe.stackdump

```diff
@@ -0,0 +1,9 @@
+Stack trace:
```

Comment:
Do we need this file in the repo? Maybe better to delete?

Reply:
Deleted